custom_metric/README.md

```diff
@@ -0,0 +1,33 @@
+# Custom Metric
```

Comment:
Can you give one or two sentences here on the purpose and/or what the sample will demonstrate?